### ISSUE TITLE
Feature Request / Image Gallery

### ISSUE THREAD

**Issue by stylefieber** [quasarframework/quasar#1786]
Hi,

since 0.15 there is (as it seems to me) no quasar image gallery.

In 0.14 we had this nice carousel with a button which shows other images. That's not possible anymore.

I would like to see some kind of image gallery, where you have an overview over all images.

**Comment by olegvakulenko** [comment]
http://quasar-framework.org/components/carousel.html

**Comment by stylefieber** [comment]
No, that's not the same. A pure carousel is not an image gallery and provides no overview over all images.

**Comment by lbssousa** [comment]
@olegvakulenko , I guess @stylefieber refer to component QGallery present in version 0.14:

http://v0-14.quasar-framework.org/components/image-gallery.html

**Comment by olegvakulenko** [comment]
Yes, now I see difference. In that case only @rstoenescu can explain this change.

**Comment by l1qu1d** [comment]
My feature request ( #1670 ) would actually solve this issue and allow high customization.

**Comment by rstoenescu** [comment]
Available in future 0.15.11
